### DISCUSSION TITLE
BlackHole, OBS and Zoom

### DISCUSSION THREAD

**Discussion by BolesBooks** [ExistentialAudio/BlackHole#599] [Q&A]
I am a new streamer using SLOBS Mac and I am grateful BlackHole exists. As I understand it, live streaming on a Mac is not really possible without BlackHole.

I have been streaming for about two weeks, and Brave browser sounds play. My streaming music from Pretzel Rocks and Monstercat play perfect. My mic, using USBPre2 is flawless.

However, when I try to use Zoom via SLOBS Mac, I have no sound. My wife is Deaf, and she teaches an ASL segment for our live stream, and pulling her into SLOBS via Zoom works great! We can see her and she can see us. All perfect; but when I begin to test a Zoom link with audio, to do a live interview with someone, there is only video and no sound. Any idea what might be happening? SLOBS support has no idea how to fix this and neither does the SLOBS community on Reddit.

VDO.NINJA works just fine for pulling in video with audio -- but the person on the other end can't see me or hear me while I'm streaming.

Is this an audio problem? A Zoom problem? Is there something else I might use that would let me do a live video interview with sound via SLOBS? One person mentioned Discord video, but I haven't been able to try that out yet because I am concerned the people I interview would have no idea what Discord, is or how to set it up, but if it is confirmed to work, then I'll make it happen. 

I thank you for your kind attention.

db

**Comment by devinroth** [comment]
I’m not familiar with SLOBS? Do you have the link to the reddit discussion?

Are you able to get BlackHole to work in other instances or programs?

**Comment by BolesBooks** [comment]
Hello!

Thanks for the help!

SLOBS is "Streamlabs OBS." They're obsessed with you, rightly so, because
without you, we Mac streamers can't stream any sound!

I was helping someone in the Reddit /streamlabs sub who wanted to move from
Windows to Mac, and I warned them against that, and Streamlabs locked the
thread:

https://www.reddit.com/r/streamlabsobs/comments/qh8f2l/migration_from_windows_to_mac/

My Zoom inquiry was actually posted in the /twitch subreddit:

https://www.reddit.com/r/Twitch/comments/qfkr0f/slobs_for_mac_and_zoom/

Every other audio instance, other than Zoom, works just fine streaming --
via SLOBS for Mac to Twitch and YouTube -- and that includes background
music.

My channel is -- https://twitch.tv/bolesbooks -- if you want to check it
out. Been using BlackHole from the start. No Zoom video/audio interviews,
though. Couldn't get it to work in testing, so didn't go livestream with
it. The most recent stream from Friday has no audio for the first 12
minutes, because I forgot to turn on my mic! Ugh!

Best,

db

_________________________________________

David Boles Books Writing & Publishing
(212) 321-3700
Boles <https://boles.com/> | Boles Bells <https://bolesbells.com/> | Boles
Books <https://bolesbooks.com/> | Boles Blogs <https://bolesblogs.com/>
Human Meme <https://itun.es/us/joZRdb.c> | Script Professor
<https://scriptprofessor.com/> | Dramatists Guild
<https://www.dramatistsguild.com/members/davidboles/> | Authors Guild
<https://www.authorsguild.net/services/members/2827>

**Comment by devinroth** [comment]
Ah of course. I guess I never heard it referred to as SLOBS before.

Post your exact setup and I will test it out when I get a chance.

**Comment by BolesBooks** [comment]
Tell me how you want me to post my exact setup.

Just list what I'm using, or do you need screenshots?

Thanks,

db

_________________________________________

David Boles Books Writing & Publishing
(212) 321-3700
Boles <https://boles.com/> | Boles Bells <https://bolesbells.com/> | Boles
Books <https://bolesbooks.com/> | Boles Blogs <https://bolesblogs.com/>
Human Meme <https://itun.es/us/joZRdb.c> | Script Professor
<https://scriptprofessor.com/> | Dramatists Guild
<https://www.dramatistsguild.com/members/davidboles/> | Authors Guild
<https://www.authorsguild.net/services/members/2827>

**Comment by devinroth** [comment]
Yeah screenshots are helpful.

**Comment by BolesBooks** [comment]
Hello!

I'm not exactly sure what screen shots you want, so here's what I came up
with:

[image: Screen Shot 2021-11-01 at 3.03.13 PM.png][image: Screen Shot
2021-11-01 at 3.03.38 PM.png]

Let me know if you need more or something else.

For the first time I ran a prerecorded video file with sound today as a
media event, and all went well. Video clear. Sound clear.

Thanks!

db

_________________________________________

David Boles Books Writing & Publishing
(212) 321-3700
Boles <https://boles.com/> | Boles Bells <https://bolesbells.com/> | Boles
Books <https://bolesbooks.com/> | Boles Blogs <https://bolesblogs.com/>
Human Meme <https://itun.es/us/joZRdb.c> | Script Professor
<https://scriptprofessor.com/> | Dramatists Guild
<https://www.dramatistsguild.com/members/davidboles/> | Authors Guild
<https://www.authorsguild.net/services/members/2827>

**Comment by devinroth** [comment]
The screenshots didn’t show up. You need to upload them via the website instead of attaching them via email.
